app/gui2/src/util/reactivity.ts

```diff
@@ -96,3 +98,37 @@ export class LazySyncEffectSet {
     this._scope.stop()
   }
 }
+
+function defaultEquality(a: unknown, b: unknown): boolean {
+  return a === b
+}
+
+/**
+ * Create a ref that is updated whenever the given function's return value changes. Similar to
+ * `computed`, but differs in significant ways:
+ * - The dependencies of the `getter` will not be propagated to any effects that access the result.
```

Comment:
TBH I don't understand this point. What are those "effects that access the result"?

Reply:
Effects, as in `watchEffect` or `render`. Anything that is created with [`effect` method from vue reactivity](https://github.com/vuejs/core/blob/main/packages/reactivity/src/effect.ts#L170-L180). Vue docs (and function names) also use that terminology.